django/contrib/staticfiles/checks.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from django.conf import settings
+from django.core.checks import Error
+
+
+def check_filesystemfinder(app_configs=None, **kwargs):
+    """
+    If using FileSystemFinder, check:
+
+    - whether STATICFILES_DIRS is a list or tuple, failing if it's not
+    - whether STATICFILES_DIRS contains a directory equivalent to STATIC_ROOT
+    """
+    errors = []
+
+    if 'django.contrib.staticfiles.finders.FileSystemFinder' in settings.STATICFILES_FINDERS:
+        if not isinstance(settings.STATICFILES_DIRS, (list, tuple)):
+            errors.append(Error(
+                'Your STATICFILES_DIRS setting is not a tuple or list; '
+                'perhaps you forgot a trailing comma?',
```

Comment:
I suggest you use the `hint` kwarg for the `'perhaps you forgot a trailing comma?'` part.